### ISSUE TITLE
Builtin IDP does not support login with email address

### ISSUE THREAD

**Issue by wkloucek** [owncloud/ocis#2572]
## Describe the bug

I can not login with the email address of my user when using the builtin IDP.

https://github.com/libregraph/lico/blob/b584e03722244bc9a20ed3eea576eaba6387e5c0/identifier/backends/ldap.go#L321

## Steps to reproduce

1. Login as `marie` -> works
2. Login as `marie@example.org`-> does not work

## Expected behavior
Login as `marie@example.org`-> also work

## Actual behavior
2. Login as `marie@example.org`-> does not work

## Setup
oCIS 1.12.0
libregraph/lico v0.34.1-0.20210803054646-b584e0372224

**Comment by pascalwengerter** [comment]
Possibly connected to #2435?

**Comment by wkloucek** [comment]
> Possibly connected to #2435?

nope... this sets the email as the username... But this was the original reason why I looked into it.
